lib/cn/intern_builder.c

```diff
@@ -92,16 +93,35 @@ struct intern_builder {
     u_char                 *sbufs;
     uint                    nodec;
     uint                    levelc;
-    struct intern_level     levelv[5];
+    struct intern_level     levelv[2];
```

Comment:
This should probably be a constant in cn_tree.h or something

Reply:
Need @gsramdasi input on this.  And how is it related to LTO?

Reply:
Do you not read comments?  The runtime data corruption issue led me here, but it turned out to be inlining of memlcp().
But before I figure that out I was looking at this code, and it's not quite right (it's correct in wbt_builder.c).  Then I noticed it was realloc'ing all the time, which defeats the point of the embedded cache, so it's now fixed to avoid that in the common case.

Reply:
So it's not related to LTO.  Could've just said that.